src/index.js

```diff
@@ -453,18 +517,24 @@ walletRouter.post('/send-tx',
       isObject: true,
       custom: {
         options: (value, { req, location, path }) => {
-          if (!('hash' in value) || !(typeof value.hash === 'string')) {
-            return false;
-          }
-          if (!('index' in value) || !(/^\d+$/.test(value.index))) {
-            // Test that index is required and it's an integer
-            return false;
+          if ('type' in value && value.type === 'query') {
```

Comment:
Missing docs explaining how to use it and what query parameters are supported. I feel that we should include at least one example.

Reply:
Added api docs for it with parameter options.

Reply:
Great! Thanks!